### ISSUE TITLE
TextFieldAssist.HasClearButton does not work on TimePicker

### ISSUE THREAD

**Issue by ProfBits** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#3135]
### Bug explanation

The clear button is visible however does not clear the field from a set time.
Minimal example xaml:
```xaml
<materialDesign:TimePicker materialDesign:TextFieldAssist.HasClearButton="True"/>
```
#1450 could be related.

### Version

4.8.0

### Edit
It's consistent in my app, the demo app I could trigger it once, need to look deeper for reporduceablility

**Comment by nicolaihenriksen** [comment]
@ProfBits Have you tried with the latest nightly build? This seems very much like a duplicate of #3119 which was recently fixed.

**Comment by ProfBits** [comment]
@nicolaihenriksen Thanks for the reply.
Yep, that was it, on tag v4.8.0 it's broken and already fixed on the current master. Sorry for not looking inside of close issues.

Closed as a Duplicat of #3119 .
